.github/workflows/arc-publish-canary.yaml

```diff
@@ -11,19 +11,19 @@ on:
       - '.github/actions/**'
       - '.github/ISSUE_TEMPLATE/**'
       - '.github/workflows/e2e-test-dispatch-workflow.yaml'
-      - '.github/workflows/e2e-test-linux-vm.yaml'
-      - '.github/workflows/publish-arc.yaml'
-      - '.github/workflows/publish-chart.yaml'
-      - '.github/workflows/publish-runner-scale-set.yaml'
-      - '.github/workflows/release-runners.yaml'
-      - '.github/workflows/run-codeql.yaml'
-      - '.github/workflows/run-first-interaction.yaml'
-      - '.github/workflows/run-stale.yaml'
-      - '.github/workflows/update-runners.yaml'
+      - '.github/workflows/gha-e2e-tests.yaml'
+      - '.github/workflows/arc-publish.yaml	'
```

Comment:
nit
```suggestion
      - '.github/workflows/arc-publish.yaml'
```

Reply:
Could you redo it please? I'm not able to merge this one cause I made a change before seeing this